metricbeat/module/kibana/stats/data.go

```diff
@@ -68,6 +68,11 @@ var (
 			"event_loop_delay": s.Object{
 				"ms": c.Float("event_loop_delay"),
 			},
+			"event_loop_utilization": c.Dict("event_loop_utilization", s.Schema{
+				"active":      c.Float("active"),
+				"idle":        c.Float("idle"),
+				"utilization": c.Float("utilization"),
+			}, c.DictOptional),
```

Comment:
The options were to mark `event_loop_utilization` as optional as I've done here, or to update `kibana.go` with special handling for versions 8.8+ so that we could conditionally include this portion of the schema only when we were on a relevant version.

I started with this approach only because it had a smaller footprint, but happy to switch it out if folks would prefer.

Reply:
Optional dictionary sounds good to me